render/render.js

```diff
@@ -347,17 +347,19 @@ module.exports = function() {
 				else if (oldStart > oldEnd) createNodes(parent, vnodes, start, end + 1, hooks, nextSibling, ns)
 				else {
 					// inspired by ivi https://github.com/ivijs/ivi/ by Boris Kaul
-					var originalNextSibling = nextSibling, vnodesLength = end - start + 1, oldIndices = new Array(vnodesLength), li=0, i=0, pos = 2147483647, matched = 0, map, lisIndices
-					for (i = 0; i < vnodesLength; i++) oldIndices[i] = -1
-					for (i = end; i >= start; i--) {
-						if (map == null) map = getKeyMap(old, oldStart, oldEnd + 1)
-						ve = vnodes[i]
-						var oldIndex = map[ve.key]
-						if (oldIndex != null) {
-							pos = (oldIndex < pos) ? oldIndex : -1 // becomes -1 if nodes were re-ordered
-							oldIndices[i-start] = oldIndex
-							oe = old[oldIndex]
-							old[oldIndex] = null
+					var originalNextSibling = nextSibling, pos = 2147483647, matched = 0
+					var oldIndices = new Array(end - start + 1)
+					for (var i = 0; i < oldIndices.length; i++) oldIndices[i] = -1
```

Comment:
Can we do this instead? Or is it slower?

```suggestion
					var oldIndices = new Int32Array(end - start + 1).fill(-1)
```

Reply:
It seems like a good idea.
Since this code path isn't tested with `npm run perf`, I'll take a quick look using dbmonster or some other benchmark.

Reply:
I tried running dbmonster, but I couldn’t tell if there was any noticeable difference. So I changed it to `new Array(end - start + 1).fill(-1)`, which reduces the bundle size.